tests/components/twitch/test_sensor.py

```diff
@@ -41,6 +41,7 @@ async def test_streaming(
     assert sensor_state.attributes["entity_picture"] == "stream-medium.png"
     assert sensor_state.attributes["game"] == "Good game"
     assert sensor_state.attributes["title"] == "Title"
+    assert sensor_state.attributes["started_at"] == "2021-03-10T03:18:11Z"
```

Comment:
When returning datetime strings in Home Assistant, they must be ISO 9601 formatted.

Ref: <https://docs.python.org/3/library/datetime.html#datetime.datetime.isoformat> including timezone information.

For example: `2024-01-01T10:10:10+01:00`

Reply:
Test has been updated. Not sure how that ever passed...